.github/workflows/deno.yml

```
@@ -6,7 +6,7 @@ jobs:
 
     strategy:
       matrix:
-        deno: [0.32.0, 0.31.0, 0.30.0]
+        deno: ["v0.83.0"]
```

There is an error in the pipeline. Is it maybe because of the `"`?
```
Error: Error: Unexpected HTTP response: 404
    at /home/runner/work/_actions/denolib/setup-deno/master/dist/index.js:3366:25
    at Generator.next (<anonymous>)
    at fulfilled (/home/runner/work/_actions/denolib/setup-deno/master/dist/index.js:3271:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

```suggestion
        deno: [0.83.0]
```

---

Well, I had the same 404 error using `[0.83.0]`, then I went on [denolib/setup-deno repository](https://github.com/denolib/setup-deno) mentioned in the article and the readme shows `deno: ["v1.0.0", "v0.42.0", "v1.x", "v0.x", "nightly"]`.
I don't really know what to do now.